docker/Dockerfile

```diff
@@ -146,8 +146,9 @@ RUN apk -U add \
         mosquitto \
         popt \
         soxr \
-        curl && \
-        rm -rfv /lib/apk/db/* && \
```

Comment:
Hi @charlesomer. I thought that running these two commands together resulted in a (slightly) smaller image, but I dunno. What do you think?